lib/manager/actions/status.js

```diff
@@ -137,44 +138,73 @@ export async function fetchAppInfo () {
   }
 }
 
+/**
+ * Constructs modal content from an service period merge (principally used for
+ * MTC).
+ */
 function getMergeFeedModalContent (result: MergeFeedsResult): ModalContent {
+  const SEPARATOR = ': '
+  const {
+    errorCount,
+    failed,
+    failureReasons,
+    idConflicts,
+    mergeStrategy,
+    remappedIds,
+    remappedReferences,
+    skippedIds
+  } = result
+  const messages = getComponentMessages('MergeFeedsResult')
   const details = []
+  details.push(
+    [messages('strategyUsed'), messages(mergeStrategy)].join(SEPARATOR)
+  )
   // Do nothing or show merged feed modal? Feed version is be created
-  details.push('Remapped ID count: ' + result.remappedReferences)
-  if (Object.keys(result.remappedIds).length > 0) {
+  details.push([messages('remappedIdCount'), remappedReferences].join(SEPARATOR))
```

Comment:
Could there be a template string used in the english.yml file instead?

Reply:
Addressed in https://github.com/ibi-group/datatools-ui/pull/686/commits/1c1239097f50b08c9c2172d9de671f91ae96cc77.